ccd-adapter/src/test/java/uk/gov/hmcts/cmc/ccd/assertion/claimantresponse/ResponseAcceptationAssert.java

```diff
@@ -16,7 +16,7 @@ public ResponseAcceptationAssert(ResponseAcceptation responseAcceptation) {
     public ResponseAcceptationAssert isEqualTo(CCDResponseAcceptation ccdResponseAcceptation) {
         isNotNull();
 
-        if (!Objects.equals(actual.getFormaliseOption().name(), ccdResponseAcceptation.getFormaliseOption().name())) {
+        if (!Objects.equals(actual.getFormaliseOption().get(), ccdResponseAcceptation.getFormaliseOption().name())) {
```

Comment:
Risk of exception. Try using `Optional.orElseThrow` instead of `Optional.get`, or get clever with filters:
```
actual.getFormaliseOption()
    .filter(option -> !Objects.equals(option, ccdResponseAcceptation.getFormaliseOption().name()))
    .ifPresent(option -> failWithMessage( ... , option));
```